supfile.go

```diff
@@ -13,21 +13,66 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+// EnvVar represents an environment variable
+type EnvVar struct {
+	Key   string
+	Value string
+}
```

Comment:
This is great @stengaard!

How does this differ from https://godoc.org/gopkg.in/yaml.v2#MapItem and https://godoc.org/gopkg.in/yaml.v2#MapSlice ?

I really appreciate your effort. We might use something similar even for the other `map[string]Command` that we use, so we could even have multiple `upload`/`run`/`local` within a command/target with a well-defined order! :)